### ISSUE TITLE
microsoft-go Azure Pipeline has red banner: "Some recent issues detected related to pipeline trigger."

### ISSUE THREAD

**Issue by dagood** [microsoft/go#320]
I'm filing this to store info about this red banner that started appearing on our `microsoft-go` AzDO build definition. As far as I can tell, it isn't affecting our builds and it's safe to ignore:

![image](https://user-images.githubusercontent.com/12819531/146243065-a2175e61-2daf-455d-8fd5-5678bf0b4031.png)

Details:

![image](https://user-images.githubusercontent.com/12819531/146243176-8a4e6175-9db5-4c93-99a1-e39c0438a379.png)

"Configuring the trigger failed, edit and save the pipeline again"

The only info I can find is someone else having the problem with their particular YAML configuration, but they also weren't seeing any adverse effects: https://stackoverflow.com/q/68819306.

**Comment by bteamsoftware** [comment]
I am seeing the same issue with a single pipeline. Builds seems to be happening fine but the error is annoying.

**Comment by dagood** [comment]
I checked our pipelines, and it looks like we aren't hitting this anymore. I searched again and found <https://developercommunity.visualstudio.com/t/Some-recent-issues-detected-related-to-p/1620293> (posted after I started this thread) which says there's been a fix. There's also some workaround and diagnosis info in the thread. Closing until we see it again.

> [@bteamsoftware] I am seeing the same issue with a single pipeline. Builds seems to be happening fine but the error is annoying.

Just to clarify: this issue was just tracking an issue in our project's pipeline, and we weren't actively trying to get AzDO to fix it. You might want to check out <https://developercommunity.visualstudio.com/t/Some-recent-issues-detected-related-to-p/1620293> and post a new thread on that board to get some eyes on your issue.
